Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1442,12 +1442,12 @@ def array(self):
 		return self._a
 
 	def isFloat(self):
-		return self._a.typecode == 'd'
+		return self._a.typecode == 'f'
```

Comment:
Humm. Do you remember how that issue was brought up to your attention? With otRound you can't observe that difference anymore. Is just a waste to use doubles there but I'm fine with switching back.

Reply:
If you flip the sign of `afloat` in the `test_double_precision_float` then it will fail, even if we use otRound as we do now:

https://github.com/fonttools/fonttools/blob/cbb33d8aa9e21f01ba72700d5748d16c09e0342b/Tests/ttLib/tables/_g_l_y_f_test.py#L165-L173

```
E       assert -242 == -243
E        +  where -243 = otRound(-242.50000000000003)
```

Besides, aren't we giving up some precision when we do arithmetic operations on floats as opposed to doubles? Or you think it'd negligible? What's the measurable effect in terms of memory/speed of using`'f'` instead of `'d'`?

Reply:
Okay I switch back to `'d`'. The effect is 8 bytes instead of 4, for every coordinate. Absolutely negligible in Python land. :)

In terms of true precision, `float` is definitely more than enough. It has 24 bits of mantissa, which means that for numbers in the 16k range (which glyph coordinates are constrained to), it has 9 bits of sub-font-unit precision.

The problem with that test is that the number you are starting with (-242.50000000000003) has more bits in it that fits in a float. The question is why does that number matter... So my question, if you remember, is how was that bug diagnosed as a bug to begin with? Where did it come from?

Does Python `float` always be double? Or is there a way to get the internal size so we match it?

Reply:
i believe python `float` is always double precision. I don't remember how the issue came up I'm afraid.

Reply:
Np. Reverted.